src/feeds/liked.feed.ts

```diff
@@ -4,7 +4,7 @@ import { LikedFeedResponseItemsItem, LikedFeedResponseRootObject } from '../resp
 
 export class LikedFeed extends Feed<LikedFeedResponseRootObject, LikedFeedResponseItemsItem> {
   @Expose()
-  private maxId: string;
+  private max_id: string;
```

Comment:
Why?

Reply:
I did not look at the other feed files, fixed so it follows the same coding standard 😄